src/components/category-box/category-box-styles.module.scss

```diff
@@ -0,0 +1,45 @@
+.box {
+  background-color: rgba(9, 20, 26, 0.45);
```

Comment:
As @kfilip94 pointed out this should have a blurred background in case the browsers supports it.
Something like this should work:
![image](https://user-images.githubusercontent.com/6906348/57760329-9c243c00-76fb-11e9-9df0-4366d81df224.png)